opensrp-chw-referral/src/main/assets/ec_client_fields.json

```diff
@@ -0,0 +1,320 @@
+          "type": "Event",
+          "json_mapping": {
+            "field": "obs.fieldCode"
+          }
+        },
+        {
+          "column_name": "base_entity_id",
+          "type": "Event",
+          "json_mapping": {
+            "field": "baseEntityId"
+          }
+        },
+        {
+          "column_name": "chw_referral_reason",
+          "type": "Event",
+          "json_mapping": {
+            "field": "obs.fieldCode"
+          }
+        },
+        {
+          "column_name": "chw_referral_service",
+          "type": "Event",
+          "json_mapping": {
+            "field": "obs.fieldCode"
+          }
+        },
+        {
+          "column_name": "chw_referral_date",
+          "type": "Event",
+          "data_type": "date",
+          "json_mapping": {
+            "field": "obs.fieldCode"
+          }
+        },
+        {
+          "column_name": "is_emergency_referral",
+          "type": "Event",
+          "json_mapping": {
+            "field": "obs.fieldCode"
+          }
+        },
+        {
+          "column_name": "danger_signs_indicator_ids",
+          "type": "Event",
+          "json_mapping": {
+            "field": "obs.fieldCode"
+          }
+        },
+        {
+          "column_name": "referral_type",
+          "type": "Event",
+          "json_mapping": {
+            "field": "obs.fieldCode"
+          }
+        },
+        {
+          "column_name": "referral_status",
+          "type": "Event",
+          "json_mapping": {
+            "field": "obs.fieldCode"
+          }
+        }
+      ]
+    },
+    {
+      "name": "ec_referral_service",
+      "columns": [
+        {
+          "column_name": "name_en",
+          "type": "Event",
+          "json_mapping": {
+            "field": "obs.fieldCode"
+          }
+        },
+        {
+          "column_name": "name_sw",
+          "type": "Event",
+          "json_mapping": {
+            "field": "obs.fieldCode"
+          }
+        },
+        {
+          "column_name": "identifier",
+          "type": "Event",
+          "json_mapping": {
+            "field": "obs.fieldCode"
+          }
+        },
+        {
+          "column_name": "is_active",
+          "type": "Event",
+          "json_mapping": {
+            "field": "obs.fieldCode"
+          }
+        }
+      ]
+    },
+    {
+      "name": "ec_referral_indicator",
+      "columns": [
+        {
+          "column_name": "name_en",
+          "type": "Event",
+          "json_mapping": {
+            "field": "obs.fieldCode"
+          }
+        },
+        {
+          "column_name": "name_sw",
+          "type": "Event",
+          "json_mapping": {
+            "field": "obs.fieldCode"
+          }
+        },
+        {
+          "column_name": "isActive",
+          "type": "Event",
+          "json_mapping": {
+            "field": "obs.fieldCode"
+          }
+        }
+      ]
+    },
+    {
+      "name": "ec_family",
```

Comment:
I don't think we need ec_family client fields on the referrals module unless its being used in any way.

Reply:
Sure, I'll eliminate the unnecessary client fields.